docker-compose-quickstart.yml

```diff
@@ -0,0 +1,31 @@
+# Inspired by https://docs.docker.com/compose/django/
+version: '3'
+
+services:
+  db:
+    # NOTE: No persistance storage configured.
+    # See: https://hub.docker.com/_/postgres/
+    image: postgres
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - ./docker-init-db.sql:/docker-entrypoint-initdb.d/init_db.sql
+
+  cache:
+    image: redis
+
+  web:
+    build: .
```

Comment:
Hmm, I think quick-start means that we will use the existing image without building it. @joeribekker is it what you wanted?

Reply:
Correct, default docker-compose is probably `build: .` and the quickstart has a pre-build image, like `image: record-management-app:latest` or so.